<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-01-17T18:33:01 IST" name="Default suite" finished-at="2024-01-17T18:33:56 IST" duration-ms="55179">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:runner.TestRunner@333d4a8c]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2024-01-17T18:33:01 IST" name="Default test" finished-at="2024-01-17T18:33:56 IST" duration-ms="55179">
      <class name="runner.TestRunner">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:runner.TestRunner@333d4a8c]" started-at="2024-01-17T18:33:01 IST" name="setUpClass" finished-at="2024-01-17T18:33:02 IST" duration-ms="811" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@e154848]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:runner.TestRunner@333d4a8c]" started-at="2024-01-17T18:33:02 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-01-17T18:33:53 IST" duration-ms="51153" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["To verify that user is able to make PNC from Paypal"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Sanity testing for Fnp.Q"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Oops! Payment not confirmed!] but found [https://uat.fnp.ae/control/checkout-page-rj#/checkout-page]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Oops! Payment not confirmed!] but found [https://uat.fnp.ae/control/checkout-page-rj#/checkout-page]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at stepdefination.FnpLogin.verify_that_user_land_on_pnc_page_as_a(FnpLogin.java:884)
at âœ½.verify that user land on PNC page as a "Oops! Payment not confirmed!"(file:///C:/Users/Aman%20Tanmay%20Varshney/Downloads/FNP_PROJECT%20-%20Aman/src/test/resources/features/fnp.feature:241)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:runner.TestRunner@333d4a8c]" started-at="2024-01-17T18:33:53 IST" name="tearDownClass" finished-at="2024-01-17T18:33:56 IST" duration-ms="3142" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- runner.TestRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
